zcutil/build.sh

```diff
@@ -46,49 +46,24 @@ Usage:
 $0 --help
   Show this help message and exit.
 
-$0 [ --enable-lcov || --disable-tests ] [ --disable-mining ] [ --enable-proton ] [ MAKEARGS... ]
+$0 [ --enable-proton ] [ MAKEARGS... ]
   Build Zcash and most of its transitive dependencies from
   source. MAKEARGS are applied to both dependencies and Zcash itself.
 
-  If --enable-lcov is passed, Zcash is configured to add coverage
-  instrumentation, thus enabling "make cov" to work.
-  If --disable-tests is passed instead, the Zcash tests are not built.
+  Pass flags to ./configure using the CONFIGURE_FLAGS environment variable.
```

Comment:
This is a user-visible incompatible change, so should be noted in the release notes.